README.md

```diff
@@ -18,97 +18,39 @@
-Atlas removes all types of tracking embedded within Windows and implements numerous group policies to minimize data collection. 
-
-Note that Atlas cannot ensure the security for things outside the scope of Windows (such as browsers and other third-party applications).
-
-### 🛡️ More Secure (over custom Windows ISOs)
-Downloading a modified Windows ISO from the internet is risky. Not only can people easily maliciously change one of the many binary/executable files included in Windows, but it also may not have the latest Windows security patches that can put your computer under serious security risks. 
-
-Atlas also comes with the following optional security features, which other custom Windows OSs strip:
-- Windows Security
-  - Windows Defender & SmartScreen (optional)
-  - Core Isolation (optional)
-  - CPU mitigations (optional)
-- Windows Updates (manual)
-
-Atlas is different. We use [AME Wizard](https://ameliorated.io) to install Atlas, and all the scripts we use are open source here in our GitHub repository. You can view the packaged Atlas playbook (`.apbx` - AME Wizard script package) as an archive, with the password being `malte` (the standard for AME Wizard playbooks), which is only to bypass false flags from antiviruses.
-
-The only executables included in the playbook are open-sourced [here](https://github.com/Atlas-OS/utilities) under [GPLv3](https://github.com/Atlas-OS/utilities/blob/main/LICENSE), with the hashes being identical to the releases. Everything else is in plain text.
-
-### ✅ Improved Performance
-Some tweaked systems on the internet have tweaked Windows too much, breaking compatibility for main features such as Bluetooth, Wi-Fi, and so on.
-Atlas is on the sweet point. It aims at getting more performance while maintaining a good level of compatibility.
-
-Some of the many changes that we have made to improve Windows are listed below:
-- Customized power scheme
-- Reduced amount of services and drivers
-- Disabled audio exclusive
-- Disabled unneeded devices
-- Disabled power saving (for personal computers)
-- Disabled performance-hungry security mitigations
-- Automatically enabled MSI mode on all devices
-- Optimized boot configuration
-- Optimized process scheduling
-
-### 🔒 Legal
-Many custom Windows OSes distribute their systems by providing a tweaked ISO of Windows. Not only does it violate [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm), but it is also not a safe way to install.
-
-Atlas partnered with the Ameliorated team to provide users with a safer and legal way to install, using [AME Wizard](https://ameliorated.io). With it, Atlas fully complies with [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm).
-
+- [Branding](https://docs.atlasos.net/branding/)
+- [FAQ & Common Issues](https://docs.atlasos.net/faq-and-troubleshooting/removed-features/)
```

Comment:
It should ideally be ordered so that:
- Installation
- FAQ & Common Issues
- Contribution Guidelines
- Branding

It's in order of importance/relevance to the user in that way